lib/generators/blacklight/templates/config/initializers/blacklight_initializer.rb

```diff
@@ -2,8 +2,4 @@
 # functionality, for example in Refworks export of Bookmarks. In Rails 4, Blacklight
```

Comment:
Isn't it already seeded? https://github.com/projectblacklight/blacklight/blob/master/lib/blacklight/engine.rb#L23-L29

So it should be safe to remove this blacklight_initalizer template then?